### ISSUE TITLE
Build fail strange

### ISSUE THREAD

**Issue by W4anD0eR96** [perl6/doc#1692]
Recent commit build fail on travis-CI with message:
```
t/pod-valid.t ........ 33/324 # Failed test 'doc/Language/operators.pod6 has clean POD6'
# at t/pod-valid.t line 33
t/pod-valid.t ........ 42/324 # Failed test 'doc/Language/regexes.pod6 has clean POD6'
# at t/pod-valid.t line 33
t/pod-valid.t ........ 119/324 # Failed test 'doc/Type/Junction.pod6 has clean POD6'
# at t/pod-valid.t line 33
t/pod-valid.t ........ 193/324 # Failed test 'doc/Type/Str.pod6 has clean POD6'
# at t/pod-valid.t line 33
t/pod-valid.t ........ 322/324 # Looks like you failed 4 tests of 324
t/pod-valid.t ........ Dubious, test returned 4 (wstat 1024, 0x400)
Failed 4/324 subtests 
t/pod6.t ............. ok
t/tabs.t ............. ok
t/typegraph.t ........ ok
Test Summary Report
-------------------
t/pod-valid.t      (Wstat: 1024 Tests: 324 Failed: 4)
  Failed tests:  34, 43, 120, 193
  Non-zero exit status: 4
```

However, `make test` with the latest version runs well locally.

```
➜  doc git:(master) make test
if [ "" != "" ]; then prove -j  -e perl6 t; else prove -e perl6 t; fi
t/00-load.t .......... ok   
t/meta.t ............. ok   
t/pod-convenience.t .. ok     
t/pod-htmlify.t ...... ok   
t/pod-valid.t ........ ok       
t/pod6.t ............. ok       
t/tabs.t ............. ok       
t/typegraph.t ........ ok   
All tests successful.
Files=8, Tests=1159, 203 wallclock secs ( 0.26 usr  0.05 sys + 305.88 cusr 30.40 csys = 336.59 CPU)
Result: PASS
```

I think this error message starts form 0a93d3f , but it seems have nothing to do with the commit.

Before it, build fails with (023ac49 ):
```
Worker information
hostname: f259353c-b17f-400b-b941-a46af37200b5@1.i-0e4d705-production-2-worker-org-ec2.travisci.net
version: v3.4.0 https://github.com/travis-ci/worker/tree/ce0440bc30c289a49a9b0c21e4e1e6f7d7825101
instance: e3d1ab8 travisci/ci-garnet:packer-1503972846 (via amqp)
startup: 4.13056152s
oci runtime error: exec failed: container_linux.go:265: starting container process caused "could not create session key: disk quota exceeded"
```

It should be mentioned that 8d7aaa5 passes the test, but, still don't know the point :-(

This prevents commits from applying, we should fix it as soon as possible.

@coke

**Comment by zoffixznet** [comment]
Considering some of the referenced pages haven't been updated in awhile, this is likely due to recent changes in Rakudo's POD handling. @tbrowder would know more

**Comment by W4anD0eR96** [comment]
Now I simply fix those tables. However, those tables should work. So @tbrowder 's commit might forget something.

The typical pattern is:
```
title1   title2
===    ====
item1  item2
item3  item4
```
above works improperly. It forces to use
```
title1 |  title2
====+====
item1 | item2
item3 | item4
```
or some other ways

**Comment by tbrowder** [comment]
On Wed, Nov 29, 2017 at 08:48 Alex Chen <notifications@github.com> wrote:

> Now I simply fix those tables. However, those tables should work. So
> @tbrowder <https://github.com/tbrowder> 's commit might forget something.
>
> The typical pattern is:
>
> title1   title2
> ===    ====
> item1  item2
> item3  item4
>
>
Without more investigation I think the above table is interpreted as a
single column table because you don’t have aligned double-space column
separators.  I’ll check further. Also read the docs on table pod to see
more details.

**Comment by tbrowder** [comment]
On Wed, Nov 29, 2017 at 9:27 AM, Tom Browder <tom.browder@gmail.com> wrote:
...
> Without more investigation I think the above table is interpreted as a
> single column table because you don’t have aligned double-space column
> separators.  I’ll check further. Also read the docs on table pod to see more
> details.

I'm correct.  See the docs.

**Comment by W4anD0eR96** [comment]
@tbrowder yes, you're right. Sorry for my boldness.

**Comment by tbrowder** [comment]
On Wed, Nov 29, 2017 at 11:22 Alex Chen <notifications@github.com> wrote:

> @tbrowder <https://github.com/tbrowder> yes, you're right. Sorry for my
> boldness.
>

No apology necessary—I didn’t mean to be so curt.

Have you found many bad tables?  I would like to see them in their original
location if possible.

**Comment by coke** [comment]
@tbrowder  this was fixed in f4426c44cfb77374c6a5e3e053b4283a4b25e345

**Comment by tbrowder** [comment]
On Wed, Nov 29, 2017 at 14:34 Will Coleda <notifications@github.com> wrote:

> @tbrowder <https://github.com/tbrowder> this was fixed in f4426c4
> <https://github.com/perl6/doc/commit/f4426c44cfb77374c6a5e3e053b4283a4b25e345>
>

Hm, those tables look ok at first glance—I’ll investigate.

**Comment by tbrowder** [comment]
On Wed, Nov 29, 2017 at 14:51 Tom Browder <tom.browder@gmail.com> wrote:

> On Wed, Nov 29, 2017 at 14:34 Will Coleda <notifications@github.com>
> wrote:
>
>> @tbrowder <https://github.com/tbrowder> this was fixed in f4426c4
>> <https://github.com/perl6/doc/commit/f4426c44cfb77374c6a5e3e053b4283a4b25e345>
>>
>
> Hm, those tables look ok at first glance—I’ll investigate.
>

After looking more closely, the changes do make more correct tables,
especially given some of the content.  There are some tables that still
need some tweaking for multi-line content in some cells, and I’ll put that
on my TODO list.

>
